build/Makefile

```diff
@@ -410,3 +418,30 @@ minikube-transfer-image:
 # Runs e2e tests against our minikube
 minikube-test-e2e: DOCKER_RUN_ARGS=--network=host -v $(minikube_cert_mount)
 minikube-test-e2e: minikube-agones-profile test-e2e 
+
+#   _____          _                  
+#  / ____|        | |                 
+# | |    _   _ ___| |_ ___  _ __ ___  
+# | |   | | | / __| __/ _ \| '_ ` _ \ 
+# | |___| |_| \__ \ || (_) | | | | | |
+#  \_____\__,_|___/\__\___/|_| |_| |_|
+ 
+setup-custom-test-cluster: $(ensure-build-image)
```

Comment:
Minor nit: for consistency sake, this should likely be called `custom-test-cluster` (custom as the prefix)

Looking back, we should be consistent with the prefixes for cluster operations - `gcloud` for GKE (or even `gke`),  `minikube` for minikube, etc. Which we _mostly_  are.

Reply:
I named it 'setup' on purpose because, opposed to the `gcloud-test-cluster` or 'minikube-test-cluster' it's not creating the cluster, but only preparing it for agones.

WDYT?

Reply:
you could make that target common to every provider and rename to reflect that and use it as a post-target in `gcloud-test-cluster` and  `minikube-test-cluster`

Reply:
ok, will try